core/objects.go

```diff
@@ -250,6 +250,12 @@ type Challenge struct {
 	R     string `json:"r,omitempty"`
 	S     string `json:"s,omitempty"`
 	Nonce string `json:"nonce,omitempty"`
+
+	// IP addresses resolved from authorization identifier during SimpleHTTP validation
+	ResolvedAddrs []net.IP `json:"resolvedAddrs,omitempty"`
+
+	// URLs redirected to during SimpleHTTP validation
+	Redirects []string `json:"redirects,omitempty"`
```

Comment:
Do the `json` directives here imply that these fields will appear in messages going back to the client?  I'm not saying that would be a bad thing, but (1) it's not in spec, and (2) it seems like there are potential privacy issues.